### PR TITLE
In this branch I build a webiste for remote course

### PR DESCRIPTION
**Pull request by MohammedElseddik**
## In this branch I : 
- created my first capstone project, the website is about remote jobs and how people can work remotely.
- created the home page and about page for mobile and desktop.
- Used a mobile-first approach to build the website and I also used a component-based design where I build the most important component to build another block of the website.
- Create the Featured section dynamically using Javascript.

### loom video link:
https://www.loom.com/share/392a02271af5458e8fd7d447e8b73eed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# HTML-CSS-JavaScript-capstone-project
\ No newline at end of file
+ Description of the project:
```
Comment (nicupop729) [inline README.md:3]: Every project should hold a descriptive title.

- [x] Kindly, ensure you have a title for your project in the README file.

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+- Major languages: HTML
+-  CSS
+    - Flexbox
+    - CSS Grid
+- JavaScript
+- Mobile first aproach
+- linter for code analysis
+
+## Deployment
+https://mohammedelseddik.github.io/Web-Development-Capstone-Project/
```
Comment (nicupop729) [inline README.md]: This link leads me to a 404 Page not found.

- [x] Please make sure that your deployment works as intended.
image

File: README.md [hunk 3]
```diff
@@ -1 +1,58 @@
...
+
+- GitHub: [@githubhandle](https://github.com/MohammedElseddik)
+- Twitter: [@twitterhandle](https://twitter.com/RebbouhElseddik)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammed-elseddik-rebbouh-676500192)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (nicupop729) [inline README.md]: Here we need to have a link to the issues page of our project. In the case of your project, the link is this one:
[issues page](https://github.com/MohammedElseddik/HTML-CSS-JavaScript-capstone-project/issues)

- [x] Please take care of this aspect, implementing the right link for issues.

File: README.md [hunk 4]
```diff
@@ -1 +1,58 @@
...
+## Built With
+
+- Major languages: HTML
+-  CSS
+    - Flexbox
+    - CSS Grid
+- JavaScript
+- Mobile first aproach
+- linter for code analysis
+
```
Comment (nicupop729) [inline README.md:16]: We should not jump over sections from the README template. Always we need to give clear instructions to users to clone our repo.

- [x] Kindly, I ask you to implement back all the sections from the README template and provide clear instructions for cloning the repo.
The link to the README template is [here](https://github.com/microverseinc/readme-template).

File: About-Page/about.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+          <span class="toggler-bar second-bar"></span>
+          <span class="toggler-bar third-bar"></span>
+        </div>
+        <ul class="list nav__list collapsible__content">
+          <li class="nav__item"><a href="../index.html">Home</a></li>
+          <li class="nav__item">
+            <a href="about.html">About</a>
+          </li>
+          <li class="nav__item"><a href="#">Programs</a></li>
+          <li class="nav__item active-item"><a href="#">Cantact Us</a></li>
```
Comment (nicupop729) [inline About-Page/about.html]: A typo slipped in here.

- [x] Please fix the typo from "Cantact Us"

File: About-Page/about.html [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&family=Lato:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../css/shared.css" />
+    <link rel="stylesheet" href="../css/about.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="header-nav">
```
Comment (nicupop729) [inline About-Page/about.html:24]: In order for our design to match the Behance design, for the top nav,

- [x] Please reduce its height;
- [x] Kindly change its colour to a darker shade;

Your design:
image
Behance design:
image

File: About-Page/about.html [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+          <li class="partner-section__list-item">
+            <i class="fab fa-apple" aria-hidden="true"> Apple</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-free-code-camp" aria-hidden="true"></i>
+            freeCodeCamp
+          </li>
+        </ul>
+      </div>
+    </section>
```
Comment (nicupop729) [inline About-Page/about.html:142]: In the Behance design, on the About page, the partners (featured Speakers on your version) are not displayed.

- [x] Please make sure to remove the partners from the About page.

Note: for the mobile version, please ensure to keep the partners (featured Speakers on your version).

File: About-Page/about.html [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+    <footer class="footer-section">
```
Comment (nicupop729) [inline About-Page/about.html:144]: In the Behance design, for the About page, the background is a shade of black.

- [x] Kindly, redesign your footer from the About page in order to have that shade of black that the Behance design is using.
Your footer:
image
Behance design:
image

**Note:** for the mobile version, please ensure to keep the white background for both Home and About pages.

File: css/home.css [hunk 9]
```diff
@@ -0,0 +1,231 @@
...
+  }
+
+  .hero__title {
+    font-size: 4rem;
+  }
+
+  .hero__description {
+    min-width: 606.438px;
+    width: 65%;
+    font-size: 2.3rem;
```
Comment (nicupop729) [inline css/home.css]: As this is a paragraph, the font size seems to be a little too big.

- [x] Let's make sure we stick to the design and rescale this description.

File: css/about.css [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+  transition: color 0.4s ease;
+}
+
+.container span a:hover {
+  color: var(--color-secondary);
+}
+
+@media screen and (min-width: 768px) {
+  .about-description {
+    font-size: 2.2rem;
```
Comment (nicupop729) [inline css/about.css]: As this description is a paragraph and we want to match the design as close as possible, it looks like it is a little to big.

- [x] Please downsize the font for this paragraph to match the Behance design

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @MohammedElseddik, 

Good job so far!
**Congratulations on your fabulous work** :champagne: :confetti_ball: 
There are some issues that you still need to work on to go to the next project but you are almost there!



### Some highlights for you
* Correct GitHubFlow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Clean codebase ✔️ 

### Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @MohammedElseddik, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**